proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -155,7 +155,7 @@ lemma invs_weak_sch_act_wf[elim!]:
 crunch tcb_at[wp]: set_endpoint "tcb_at t"
 crunch tcb_at'[wp]: setEndpoint "tcb_at' t"
 
-lemma blocked_cancel_ipc_corres:
+lemma blocked_cancelIPC_corres:
```

Comment:
what I said in the mattermost verification channel was actually `blockedCancelIPC_corres`, but I realise that this is what MCS has, and in master something similar is called `blockedIPCCancel`. So, maybe this one should be `blockedIPCCancel_corres`?? What’s in MCS is quite new, in fact, added by Ed last year. I am not sure if we like such inconsistency, but maybe we can just live with it for this case inside `cancelIPC`, given that the function for the reply case will definitely have different names anyway.